src/App.js

```diff
@@ -1,49 +1,96 @@
+/* global fetch */
+
 import React, { Component } from 'react'
-import {
-  FormGroup,
-  FormControl
-} from 'react-bootstrap'
 import './App.css'
-import PasteInlineForm from './pasteInlineForm.js'
+import PastePage from './PastePage'
+import {
+  BrowserRouter as Router,
+  Route
+} from 'react-router-dom'
 
-class App extends Component {
-  constructor (props) {
-    super(props)
+class TagsPage extends Component {
+  render () {
+    return (
+      <div>
+        <p>the tags page</p>
+      </div>
+    )
+  }
+}
+
+class DisplayPage extends Component {
+  constructor () {
+    super()
 
     this.state = {
-      codeText: '',
-      codeTitle: '',
-      codeAuthor: ''
+      codeText: ''
+    }
+  }
+
+  ifFbKeyOrNot (aStr) {
```

Comment:
you might use `db` (as in database) instead of `fb` in your code so as to remove the smell of facebook